data/source/database/adapter/MySql.php

```diff
@@ -320,6 +320,11 @@ protected function _execute($sql, array $options = array()) {
 			$options = $params['options'];
 			$conn->setAttribute(PDO::MYSQL_ATTR_USE_BUFFERED_QUERY, $options['buffered']);
 
+			if (strpos($conn->getAttribute(PDO::ATTR_CLIENT_VERSION), 'mysqlnd') !== false) {
+				$conn->setAttribute(\PDO::ATTR_EMULATE_PREPARES, false);
```

Comment:
According to http://blog.ulf-wendel.de/2008/pdo_mysqlnd-the-new-features-of-pdo_mysql/

> mysqlnd returns native data types when using Server-side Prepared Statements, for example an INT column is returned as an integer variable not as a string. 

My initial tests illustrated that `$conn->setAttribute(\PDO::ATTR_STRINGIFY_FETCHES, false);` worked without `ATTR_EMULATE_PREPARES`. However, everything I could find on the topic suggested including it.